From 648bc44c529e8e69b14c17d46bc960cf44b950c1 Mon Sep 17 00:00:00 2001
From: Hans-Kristian Arntzen <post@themaister.net>
Date: Wed, 20 Aug 2025 22:35:56 +0200
Subject: [PATCH] Enable JNI/mediacodec.

---
 scripts/ffmpeg/build.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/ffmpeg/build.sh b/scripts/ffmpeg/build.sh
index 620d1c9..a116853 100755
--- a/scripts/ffmpeg/build.sh
+++ b/scripts/ffmpeg/build.sh
@@ -45,6 +45,8 @@ EXTRA_LDFLAGS="-Wl,-z,max-page-size=16384 $DEP_LD_FLAGS"
   --strip=${FAM_STRIP} \
   --extra-cflags="-O3 -fPIC $DEP_CFLAGS" \
   --extra-ldflags="$EXTRA_LDFLAGS" \
+  --enable-jni \
+  --enable-mediacodec \
   --enable-shared \
   --disable-static \
   --disable-vulkan \
-- 
2.50.1

